flutter/android/docker/Dockerfile

```diff
@@ -75,3 +75,4 @@ RUN chmod -R 754 $HOME
 RUN mkdir -p /mnt/cache && chmod 777 /mnt/cache
 RUN mkdir -p /mnt/project/flutter/build && chmod 777 /mnt/project/flutter/build
 RUN mkdir -p $HOME/.cache/bazelisk && chmod 777 $HOME/.cache/bazelisk
+ENV FIREBASE_PROJECT_ID=0
```

Comment:
@anhappdev this is the build id i am talking about in the previous comment.

Reply:
Did you set up Firebase env vars as described here?
https://github.com/mlcommons/mobile_app_open/blob/master/docs/environment-setup/env-setup-firebase.md

Reply:
You don’t need to add this line.